clients/fides-js/src/components/tcf/TcfOverlay.tsx

```diff
@@ -87,12 +91,51 @@ export interface UpdateEnabledIds {
   modelType: keyof EnabledIds;
 }
 
+type Category =
+  | "purpose_consent"
+  | "purpose_legitimate_interests"
+  | "special_feature"
+  | "vendor_consent"
+  | "vendor_legitimate_interests"
+  | "system_consent"
+  | "system_legitimate_interests";
+type NoticeSubMap = {
+  [key: string]: string | number;
+};
+
+type NoticeMap = Record<Category, NoticeSubMap>;
+
+const noticeMap = (servedNotices: LastServedConsentSchema[]): NoticeMap => {
```

Comment:
heh I guess these shouldn't really be called `servedNotices` either (you probably noticed the backend type changed from `LastServedNoticeSchema` --> `LastServedConsentSchema`) so I guess this should be `servedConsent`. it's definitely a nit, but I think not conflating notices and TCF is a good distinction that helps with understanding all that's going on here